helm/thingsboard/templates/initializedb-job.yaml

```diff
@@ -27,7 +27,14 @@ spec:
       imagePullSecrets:
         - name: '{{ .Release.Name }}-container-registry'
       initContainers:
-        - name: check-db-ready
+        {{- if .Values.cassandra.enabled }}
+        - name: check-cassandra-db-ready
+          image: 'cassandra:3.11'
```

Comment:
Nit: why is this tag hardcoded here? Can we use same tag as cassandra chart?

Reply:
I haven't found a good way for getting this value from another variable in the chart.
I tryied to use the bitnami image but it complains with an error. 
Let's keep it hardcoded for now as it is in thingsboard-private repository.